pkg/apis/policies/v1/webhook_suite_test.go

```diff
@@ -581,11 +582,21 @@ var _ = Describe("validate PolicyServer webhook with ", func() {
 		Expect(err).NotTo(HaveOccurred())
 
 		By("checking default values")
+		fmt.Fprint(GinkgoWriter, pol.ObjectMeta.Finalizers)
```

Comment:
I think this is a debug statement that should be removed. Isn't it? :)

Reply:
woops! thanks